### PR TITLE
Abw 2598 UI teaks main seed phrase required

### PR DESCRIPTION
**Pull request by CyonAlexRDX**
[Jira ABW-2598](https://radixdlt.atlassian.net/browse/ABW-2598)

## Before
IMG_0669

## After

IMG_BC4F5A515985-1

### CODE REVIEW

File: RadixWallet/Features/ProfileBackupsFeature/RestoreProfileFromBackup/Children/ImportSeedPhrasesFlow/ConfirmSkippingBDFS/ConfirmSkippingBDFS+View.swift [hunk 1]
```diff
@@ -21,13 +21,14 @@ extension ConfirmSkippingBDFS {
 		public var body: some SwiftUI.View {
 			WithViewStore(store, observe: \.viewState, send: { .view($0) }) { viewStore in
 				VStack(spacing: .medium2) {
-					// FIXME: Strings
-					Text("No Main Seed Phrase?")
+					Text(L10n.RecoverSeedPhrase.Header.titleNoMainSeedPhrase)
 						.textStyle(.sheetTitle)
-						.padding(.horizontal, -.small2)
```
Comment (kugel3) [inline RadixWallet/Features/ProfileBackupsFeature/RestoreProfileFromBackup/Children/ImportSeedPhrasesFlow/ConfirmSkippingBDFS/ConfirmSkippingBDFS+View.swift:27]: Nice to see this go

File: RadixWallet/Features/ProfileBackupsFeature/RestoreProfileFromBackup/Children/ImportSeedPhrasesFlow/ConfirmSkippingBDFS/ConfirmSkippingBDFS+View.swift [hunk 2]
```diff
@@ -21,13 +21,14 @@ extension ConfirmSkippingBDFS {
...
+						.foregroundColor(.app.gray1)
+						.multilineTextAlignment(.center)
 
 					ScrollView(.vertical, showsIndicators: true) {
-						Text("The Radix Wallet always uses a single main “Babylon” seed phrase to generate new Personas and new Accounts (when not using a Ledger device).\n\nIf you do not have access to your previous main seed phrase, you can skip entering it for now. The Radix Wallet will create a new one, which will be used for new Personas and Accounts.\n\nYour old Accounts and Personas will still be listed, but you will have to enter their original seed phrase to use them. Alternatively, you can hide them if you no longer are interested in using them.")
+						// we use `.init` to preserve bold attribute of string.
+						Text(.init(L10n.RecoverSeedPhrase.Header.subtitleNoMainSeedPhrase))
```
Comment (kugel3) [inline RadixWallet/Features/ProfileBackupsFeature/RestoreProfileFromBackup/Children/ImportSeedPhrasesFlow/ConfirmSkippingBDFS/ConfirmSkippingBDFS+View.swift:31]: I would prefer `LocalizedStringKey` to `.init`, and a less cryptic comment but you will probably object on the grounds of CI being slow...

**Review by kugel3 — APPROVED** [review]

File: RadixWallet/Features/ProfileBackupsFeature/RestoreProfileFromBackup/Children/ImportSeedPhrasesFlow/ConfirmSkippingBDFS/ConfirmSkippingBDFS+View.swift (hunk 2 above)
Comment (CyonAlexRDX) [reply RadixWallet/Features/ProfileBackupsFeature/RestoreProfileFromBackup/Children/ImportSeedPhrasesFlow/ConfirmSkippingBDFS/ConfirmSkippingBDFS+View.swift]: I changed to `LocalizedStringKey` in 3 files.

Comment (kugel3) [reply RadixWallet/Features/ProfileBackupsFeature/RestoreProfileFromBackup/Children/ImportSeedPhrasesFlow/ConfirmSkippingBDFS/ConfirmSkippingBDFS+View.swift]: The typechecker and I thank you

**Pull request merged**